### DISCUSSION TITLE
Difference between `vimium-c` & `vimium` ?

### DISCUSSION THREAD

**Discussion by ghost** [gdh1995/vimium-c#731] [Q&A] [answered]
Is there a wiki section to read differences between this & https://github.com/philc/vimium ?

**Comment by gdh1995** [comment, marked as answer]
I once wrote an answer: https://github.com/gdh1995/vimium-c/issues/12#issuecomment-344797440  , but it lacks many new features of Vimium C . Maybe release notes help: https://github.com/gdh1995/vimium-c/blob/master/RELEASE-NOTES.md
